forage-android/src/main/java/com/joinforage/forage/android/ui/ForageElement.kt

```diff
@@ -5,26 +5,27 @@ import com.joinforage.forage.android.core.element.SimpleElementListener
 import com.joinforage.forage.android.core.element.StatefulElementListener
 import com.joinforage.forage.android.core.element.state.ElementState
 
-data class ForageContext(
+/**
+ * Configuration details required by ForageElement to operate correctly
+ *
+ * @property merchantId Your merchant FNS number
+ * @property sessionToken A temporary token used for authenticate
+ */
+data class ForageConfig(
     val merchantId: String,
     val sessionToken: String
-    // TODO: thoughts on customerId being part of the
-    //  ForageContext instead of a param for tokenizeEBTCard?
-    //  Is there a future where we use customerID as part of
-    //  fraud detection in checkBalance?
 )
 
-// an interface that represents that abstract state of
-// a ForageElement.
-// NOTE: isValid is true when validationError = null
-
-// Minimalist event signatures where the only information
-// conveyed is whether or not a specific event has occurred
 
+/**
+ * The interface that all Forage UI Components adhere to. For example,
+ * both the ForagePANEditText and the ForagePINEditText satisfy
+ * ForageElements
```

Comment:
```suggestion
 * ForageElement
```

Reply:
🤦 nice one

Reply:
address in [1cfd621](https://github.com/teamforage/forage-android-sdk/pull/97/commits/1cfd621c57326881b573e5f5878edb556cf963d3)